app/features/receive/cashu-receive-quote-repository.ts

```diff
@@ -210,10 +215,7 @@ export class CashuReceiveQuoteRepository {
     }
 
     const [updatedQuote, updatedAccount] = await Promise.all([
-      CashuReceiveQuoteRepository.toQuote(
-        data.updated_quote,
-        this.encryption.decrypt,
-      ),
+      CashuReceiveQuoteRepository.toQuote(data.updated_quote),
```

Comment:
this is not a promise at all anymore so we can just do a normal call and then await AccountRepository.toAccount and get rid of Promise.all